andela/anbu-cfh
# Feature/134249279/users invite max min

#### What does this PR do?
Implements Display of pop ups upon max or min number of players conditions.

#### Description of Task to be completed?
User can click the start game button but a dialog pops up if number of players less than the required minimum and if No of invites sent exceeds maximum limit of players

#### How should this be manually tested?
Click the Start game button with less than 3 players.
Invite Players using the Invite Players link. Should also warn you upon reaching limit

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
##134249279- Users should be able to send invites to at most 11 players other than themselves
#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
